src/10_functions.py

```diff
@@ -1,12 +1,26 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(n):
```

Comment:
You can get away with not doing an else return False here since it will implicitly return False unless specified as True
```
def is_even(num):
    if num % 2 == 0:
        return True
```